postgres/supabase/supabase.md

```diff
@@ -0,0 +1,20 @@
+### Follow these steps to get started with Supabase as a Data Accelerator.
+
+**Step 1.** Create a Supabase account and project [https://supabase.com/](https://supabase.com/). Supabase will set up a new database as part of the account/project creation. Refer to the [Supabase Documentation](https://supabase.com/docs/guides/database/overview) for creating new databases in an existing project.
+
+**Step 2.** Open the newly created Supabase project and select `Project Settings` from the sidebar navigation.
+
+**Step 3.** Navigate to `Database` under `Configuration`.
+
+**Step 4.** Find the `Connection parameters` section.
+![Connection parameters](https://imagedelivery.net/HyTs22ttunfIlvyd6vumhQ/02a30c41-072c-49cc-cba3-e29f35ca6800/public)
+
+**Step 5.** Edit the `spicepod.yaml` file in this directory and replace the params in the `eth_recent_blocks` dataset with the connection parameters from the Supabase project. Run the Spice.ai runtime with `spice run` from this directory.
```

Comment:
Can we replace this with a `spice add spiceai/supabase-quickstart`?

Reply:
Since this provides configuration template that needs to be replaced with the user's supabase connection values it doesn't feel like it belongs in spice rack. As a user I would expect anything I pull from spice rack to be valid YAML and ready to use out of the box.

Reply:
Yeah, make sense.